omero/sysadmins/unix/install-web.txt

```diff
@@ -16,37 +16,41 @@ with:
 -  the built-in Django lightweight development server (for **testing**
    only; see the :doc:`/developers/Web/Deployment` page)
 
-If you need help configuring your firewall rules, see the
-:doc:`/sysadmins/server-security` page.
+If you need help configuring your firewall rules, see
+:doc:`/sysadmins/server-security` for more details.
 
 
 .. _omero_web_deployment:
 
 Deployment
 ----------
 
-There are a number of good ways to easily deploy OMERO.web.
-
-Deploying OMERO.web separately from OMERO.server is recommended as they
+Deploying OMERO.web **separately** from OMERO.server is **recommended** as they
 perform best under different circumstances and require a different set of
-dependencies. Please consider using :doc:`OMERO.web WSGI deployment <install-web/web-deployment>`.
+dependencies. See :doc:`install-web/web-deployment` for more details.
 
 If you have installed Nginx, OMERO can automatically generate a
 configuration file for your web server. The location of the file will depend
 on your system, please refer to your web server's manual. See
 :ref:`customizing_your_omero_web_installation` for additional
-customization options.
+customization options. Below is a list of OS specific walkthroughs.
```

Comment:
I think it would probably be clearer is the 'Below is a list...' sentence was a new paragraph (below the hidden toctree) and maybe an extra few words of explanation would be helpful